src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,55 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int OPERATION_INDEX = 0;
+    private static final int AMOUNT_INDEX = 1;
+    private static final String SUPPLY = "supply";
+    private static final String COMA = ",";
+
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
The getStatistic() method should ideally consist of only 3 method calls:
String data = readFromFile(fromFileName);
String report = createReport(data);
writeToFile(data, toFileName);